happi/cli.py

```diff
@@ -64,9 +64,12 @@ def happi_cli(ctx, path, verbose):
               help='Show results in JSON format.')
 @click.option('--names', is_flag=True,
               help='Return results as whitespace-separated names.')
+@click.option('--glob/--regex', 'use_glob', default=True,
+              help='use glob style (default) or regex style search terms. '
+              'Regex requires backslashes to be escaped (eg. at\\\\d.\\\\d)')
```

Comment:
Raw string to lessen the escape chars, perhaps?
```suggestion
              r"Regex requires backslashes to be escaped (eg. at\d.\d)"
```

Reply:
Also I like that `click.option` can easily specify the mutually-exclusive flags here. Neat!